app/lib/backend_api_logic/routing_policy.rb

```diff
@@ -10,10 +10,17 @@ def policy_chain
       other_routing_rules = other_routing_policies.flat_map { |policy| policy['configuration']['rules'] }
 
       routing_policy = Builder.new(service).to_h
-
+      
```

Comment:
Try setting up the auto-removal of empty white-spaces in your IDE to avoid this.